Khrystyna_Derenivska/src/app/app.component.html

```diff
@@ -0,0 +1,9 @@
+<!-- <app-new-item-add #appChild></app-new-item-add> -->
+<div class="new-item-add">
+    <label for="new-task-name">Add new task</label>
+    <input type="text" id="add-task-name" [(ngModel)]='taskName'>
+    <button class="add-button" (click)="onCreateButtonClick()">Add</button>
+</div>
+
+<hr>
+<app-task-item *ngFor="let task of tasks" [task]='task' (deleteItem)="onDelete($event)"></app-task-item>
```

Comment:
Why you add 'on' to name of method?

Reply:
I've used to do that when I write event handler function